bin/csv_to_krun_json

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env pypy
+"""
+Convert a CSV file of timing measurements to JSON, suitable for using with the
+other scripts here.
+"""
+
+import argparse
+import csv
+import os
+import os.path
+import sys
+
+sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', 'warmup'))
+from warmup.krun_results import write_krun_results_file
+
+
+def main(in_files, language, vm, uname):
+    for filename in in_files:
+        data_dictionary = { 'data': dict(), # Measurement data.
+                   'core_cycles_data': dict(),
+                   'aperf_data': dict(),
+                   'mperf_data': dict(),
+                   'instr_data': dict(),
+                   'audit': dict(),
+                   'config': '',
+                   'reboots': 0,
+                   'starting_temperatures': list(),
+                   'eta_estimates': list(),
+                   'error_flag': list(), }
+        with open(filename, 'r') as fd:
+            data_dictionary['audit']['uname'] = uname
+            reader = csv.reader(fd)
+            firstrow = True
```

Comment:
Wouldn't it be easier to say something like `reader.next()` (or `reader.readline()` or whatever it's called) to munch the first line, rather than having to have a variable and an `if` check inside the `for` loop?